astropy/time/formats.py

```diff
@@ -710,38 +710,48 @@ class TimeUnix(TimeFromEpoch):
 
 class TimeUnixTai(TimeUnix):
     """
-    Seconds from 1970-01-01 00:00:08 TAI (see notes), including leap seconds.
+    Unix time (TAI): SI seconds elapsed since 1970-01-01 00:00:00 TAI (see caveats).
 
-    This will generally differ from Unix time by the cumulative integral number
-    of leap seconds since 1970-01-01 UTC.  This convention matches the definition
-    for linux CLOCK_TAI (https://www.cl.cam.ac.uk/~mgk25/posix-clocks.html).
+    This will generally differ from standard (UTC) Unix time by the cumulative
+    integral number of leap seconds introduced into UTC since 1972-01-01 UTC
+    plus the initial offset of 10 seconds at that date.  To date, 27 positive
+    leap seconds were introduced into UTC, resulting in the current offset of
```

Comment:
This might become wrong in the future, so I'd just cut this `To date` sentence.

Reply:
Removed and added comments and another example for the offset in 1972.